src/components/Footer/index.js

```diff
@@ -16,6 +18,8 @@ import {
   Wrapper
 } from './styles'
 
+const docsPage = getFirstPage()
```

Comment:
means that `https://dvc-landing-gatsby-migr-abefpq.herokuapp.com/doc/` does not work

Reply:
We need to add it to the redirects list, yes. Having two identical pages on different urls probably not good for SEO too.

Reply:
Added redirects for all pages with `source: false`.

Reply:
> probably not good for SEO too.

yes, it's pretty bad

> We need to add it to the redirects list

make sure that it is 302 or 307 or whatever we use for temporary ones - we don't want them to be cached. we need to have a separate index page for dvc.org/doc

Reply:
Added 307 status code.